spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestAddFilesProcedure.java

```diff
@@ -830,6 +830,44 @@ public void invalidDataImportPartitioned() {
         .hasMessageContaining("that matches the partition columns");
   }
 
+  @TestTemplate
+  public void partitionColumnCountMismatchInFilter() {
+    createPartitionedHiveTable();
+
+    createIcebergTable(
+        "id Integer, name String, dept String, subdept String", "PARTITIONED BY (id)");
+    assertThatThrownBy(
+            () ->
+                scalarSql(
+                    "CALL %s.system.add_files(table => '%s', source_table => '%s', partition_filter => map('id', '0','dept', '1'))",
```

Comment:
nti: We can omit parameter names (table, source_table, partition_filter). Same for another test.

Reply:
done